uaclient/entitlements/repo.py

```diff
@@ -297,3 +297,7 @@ def remove_apt_config(self):
                 )
             elif os.path.exists(repo_pref_file):
                 os.unlink(repo_pref_file)
+        print("Updating package lists")
```

Comment:
Let's pull this text up into a uaclient/status.py message in the event that we internationalize messaging at some point.
We also have this same message emitted on line 254  in setup_apt_config so we can use that status message there too.